### PR TITLE
About page layout updates

### PR DESCRIPTION
**Pull request by clhenrick**
Implements layout & design changes as outlined in the [Design Audit doc – about section](https://docs.google.com/document/d/1SbYxkLkx9RAuhiCb_NJ0d06weKwdia70zg1TYoN6GcI/edit#bookmark=id.iixv69tit2d6)

Bonus changes for About & other top level pages:

- For top level pages that have a banner image I cropped and reduced their file sizes since many of them were about or over 1MB in size.
- Most banner images have a mobile and > mobile version and are toggled via CSS media queries

### CODE REVIEW

**Comment by clhenrick** [comment]
[About](https://trello.com/c/Ddp9G4KD/7-about)

**Comment by clhenrick** [comment]
@mukhtyar [deploy preview here](https://614e56c4a494f611ea916617--happy-engelbart-4042ee.netlify.app/about/)

File: src/routes/about/index.svelte [hunk 1]
```diff
@@ -42,36 +50,78 @@
 </script>
 
 <style lang="scss">
+  figure {
+    margin: 0;
+
+    @media (max-width: 1052px) {
```
Comment (clhenrick) [inline src/routes/about/index.svelte:56]: Note: these hard-coded breakpoints in pixels correspond to the Carbon Grid breakpoints.

It's annoying that you can't currently use CSS variables with media queries, although supposedly a future version of media queries will support using them. In the meantime I'm wondering if they are exposed in the Carbon Design system Sass?

**Review by mukhtyar — APPROVED** [review]
The About page looks great!

**Pull request merged**